OCSP staping legacy support (#49)
lamps-wg/RFC5019bis: draft-ietf-lamps-rfc5019bis.md

Further information regarding caching issues can be obtained
from {{?RFC3143}}.

To provide support to OCSP clients which do not yet
support the use of SHA-256 for CertID hash calculation, the OCSP

> responder MAY include two SingleResponses in the OCSP basic response.

Reviewer: ```suggestion
responder MAY include two SingleResponses in a BasicOCSPResponse.
```

> The CertID of one of the SingleResponses uses SHA-1 for the hash

Reviewer: ```suggestion
In that BasicOCSPResponse, the CertID of one of the SingleResponses uses SHA-1 for the hash
```
